app/assets/javascripts/discourse/app/form-kit/components/fk/control/select.gjs

```diff
@@ -1,46 +1,29 @@
 import Component from "@glimmer/component";
 import { hash } from "@ember/helper";
-import { on } from "@ember/modifier";
-import { action } from "@ember/object";
-import { NO_VALUE_OPTION } from "discourse/form-kit/lib/constants";
-import { i18n } from "discourse-i18n";
-import FKControlSelectOption from "./select/option";
+import DSelect, { DSelectOption } from "discourse/components/d-select";
+
+const SelectOption = <template>
+  <DSelectOption
+    @value={{@value}}
+    @selected={{@selected}}
+    class="form-kit__control-option"
```

Comment:
I did this because I could figure out how to pass class attribute using `(component ComponentName)` syntax